src/main/java/uk/gov/pay/api/model/RequestError.java

```diff
@@ -86,7 +86,8 @@ public enum Code {
         GET_AGREEMENT_LEDGER_ERROR("P0298", "Downstream system error"),
 
         CANCEL_AGREEMENT_CONNECTOR_BAD_REQUEST_ERROR("P0501", "Cancellation of agreement failed"),
-        CANCEL_AGREEMENT_CONNECTOR_ERROR("P0198", "Downstream system error");
+        CANCEL_AGREEMENT_CONNECTOR_ERROR("P0198", "Downstream system error"),
+        SEARCH_DISPUTES_VALIDATION_ERROR("P0401", "Invalid parameters: %s. See Public API documentation for the correct data formats");
```

Comment:
Would it make sense to put this next to the other error that uses code P0401 just so it's easier to see that this multiple use case exists.

Or as the message is the same, use the existing Enum value, possibly renaming it to make it more generic?

Reply:
I'll move it in next iteration 👍